src/99-appendices/03-hed.md

```diff
@@ -45,35 +45,73 @@ Example:
 ```Text
 onset	duration	mycodes
 1.1	n/a	Fixation  
-1.3	n/a	Button
-1.8	n/a	Target
+1.3	n/a	Button  
+1.8	n/a	Target  
 ...
 
 ```
+ 
+The tags in the `HED` column of the \_events.tsv file are often specific to the individual event instances,
```

Comment:
Please used code formatting with the markdown "backtick" syntax instead of escape characters:
````
\_events.tsv --> `_events.tsv`
````


```suggestion
The tags in the `HED` column of the `*_events.tsv` file are often specific to the individual event instances,
```

Reply:
Done